Makefile

```diff
@@ -20,7 +20,7 @@ WEBLATE_REPO=tmp/weblate-repo
 WEBLATE_REPO_URL=https://github.com/cockpit-project/cockpit-machines-weblate.git
 WEBLATE_REPO_BRANCH=main
 
-PYEXEFILES=$(shell git grep -lI '^#!.*python')
+PYEXEFILES=$(shell git grep -lI '^#!.*python') $(shell find test/ -name *.py ! -path 'test/common/*')
```

Comment:
Seems a bit weird to call these `PYEXEFILES`. Either call these `PYTHONFILES` and use `git ls-tree -r --name-only HEAD | grep '\.py'` (to really only catch the tracked files, and avoid doing this `! -path` stuff) or call pyflakes/pycodestyle without arguments to let them find all the py files (not 100% sure that this works).